relayer/relays/beacon/cache/cache.go

```diff
@@ -22,10 +22,6 @@ type Finalized struct {
 	LastSyncedHash common.Hash
 	// Stores the last successfully synced slot
 	LastSyncedSlot uint64
-	// Stores the last attempted finalized header, whether the import succeeded or not.
-	LastAttemptedSyncHash common.Hash
-	// Stores the slot number of the above header
-	LastAttemptedSyncSlot uint64
```

Comment:
Remove these attributes since we've introduced cross-check so sync `FinalizedUpdate` retry with same slot is nessessary.